_includes/header.html

```diff
@@ -16,7 +16,18 @@
       <ul class="nav navbar-nav navbar-right navbar-collapse">
         <li id="home" {% if page.url == "/" %} class="active" {% endif %}><a href="/">Catalog</a></li>
         <li id="news" {% if page.url == "/news/" %} class="active" {% endif %}><a href="/news/">News</a></li>
-        <li id="about" {% if page.url == "/about/" %} class="active" {% endif %}><a href="/about/">About</a></li>
+        <li id="about"
+            {% if page.url == "/about/" %} class="active dropdown"
+            {% else %} class="dropdown"
+            {% endif %}><a href="/about/">About</a>
+          <div class="dropdown-content">
+            <p><a href="about/using-github.html">Using GitHub</a></p>
```

Comment:
Should these have a leading `/` and NOT have the trailing `.html` ?

Looks like that is how we have the other links on the site...

Reply:
thanks for the catch. I didn't do a full build locally.